app/assets/javascripts/controllers/miq_ae_class/ae_method_form_controller.js

```diff
@@ -0,0 +1,263 @@
+aeMethodFormController.$inject = ['$http', '$scope', 'aeMethodFormId', 'currentRegion', 'miqService', 'playbookReusableCodeMixin'];
+
+function aeMethodFormController($http, $scope, aeMethodFormId, currentRegion, miqService, playbookReusableCodeMixin) {
+  var vm = this;
+  var init = function() {
+    vm.aeMethodModel = {
+      name: '',
+      display_name: '',
+      location: '',
+      namespace_path: '',
+      class_id: '',
+      language: '',
+      scope: '',
+      key: '',
+      key_value: '',
+      key_type: 'string',
+      available_datatypes: '',
+      provisioning_repository_id: '',
+      provisioning_playbook_id: '',
+      provisioning_machine_credential_id: '',
+      provisioning_network_credential_id: '',
+      provisioning_cloud_credential_id: '',
+      provisioning_key: '',
+      provisioning_value: '',
+      provisioning_type: 'string',
+      provisioning_inputs: [],
+      provisioning_verbosity: '0',
+      provisioning_editMode: false,
+      provisioning_become_enabled: false,
+    };
+    vm.verbosity_types = playbookReusableCodeMixin.getVerbosityTypes();
+    vm.provisioning_cloud_type = '';
+    vm.currentRegion = currentRegion;
+    vm.formId = aeMethodFormId;
+    vm.afterGet = false;
+    vm.model = "aeMethodModel";
+
+    ManageIQ.angular.scope = $scope;
+
+    $http.get('method_form_fields/' + aeMethodFormId)
+      .then(getMethodFormData)
+      .catch(miqService.handleFailure);
+    vm.saveable = miqService.saveable;
+    vm.newRecord = aeMethodFormId === 'new';
+    if (aeMethodFormId === 'new') {
+      playbookReusableCodeMixin.formOptions(vm);
+    }
+  };
+
+  function getMethodFormData(response) {
+    var data = response.data;
+    vm.aeMethodModel.name = data.name;
+    vm.aeMethodModel.display_name = data.display_name;
+    vm.aeMethodModel.namespace_path = data.namespace_path;
+    vm.aeMethodModel.location = data.location;
+    vm.aeMethodModel.class_id = data.class_id;
+    vm.aeMethodModel.language = data.language;
+    vm.aeMethodModel.scope = data.scope;
+    vm.aeMethodModel.available_datatypes = data.available_datatypes;
+    playbookReusableCodeMixin.formOptions(vm);
+    playbookReusableCodeMixin.formCloudCredentials(vm, data.config_info.cloud_credential_id, null);
+    getConfigInfo(data.config_info);
+    vm.modelCopy = angular.copy(vm.aeMethodModel);
+  }
+
+  var getConfigInfo = function(configData) {
+    vm.aeMethodModel.provisioning_repository_id = configData.repository_id;
+    vm.aeMethodModel.provisioning_playbook_id = configData.playbook_id;
+    vm.aeMethodModel.provisioning_machine_credential_id = configData.credential_id;
+    vm.aeMethodModel.provisioning_network_credential_id = configData.network_credential_id;
+    vm.aeMethodModel.provisioning_cloud_credential_id = playbookReusableCodeMixin.setIfDefined(configData.cloud_credential_id);
+    vm.aeMethodModel.provisioning_become_enabled = configData.become_enabled === 'true' ? true : false;
```

Comment:
`vm.aeMethodModel.provisioning_become_enabled = configData.become_enabled === 'true'` is equal to 
`vm.aeMethodModel.provisioning_become_enabled = configData.become_enabled === 'true' ? true : false;`